smarts/core/waypoints.py

```diff
@@ -112,6 +112,7 @@ def __eq__(self, other):
         "wp",  # Waypoint: current waypoint
         "nexts",  # list of LinkedWaypoint: list of next immediate waypoints
         # it's a list of waypoints because a path may branch at junctions
+        "is_shape_wp",
```

Comment:
Is it possible to assign a default value?

Reply:
Thanks, sure.